packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -905,8 +905,7 @@ test('+server.js next to +page.svelte works', async ({ page }) => {
 	expect(await page.textContent('p')).toBe('Hi');
 
 	for (const method of ['GET', 'PUT', 'PATCH', 'POST', 'DELETE']) {
-		page.click(`button:has-text("${method}")`);
-		await page.waitForResponse('/routing/endpoint-next-to-page');
-		expect(await page.textContent('pre')).toBe(method);
+		await page.click(`button:has-text("${method}")`);
+		await expect(page.locator('pre')).toHaveText(method);
```

Comment:
This is basically how it looked like prior to Rich's change, which is why I'm so confused why it would be flaky in the first place
https://github.com/sveltejs/kit/commit/2af2a9dba148ee8403ef5c163ef8827eb27a4800